extension/js/common/composer.ts

```diff
@@ -299,9 +299,9 @@ export class Composer {
     }, this.getErrHandlers(`change footer`)));
     $('.delete_draft').click(Ui.event.handle(async () => {
       await this.draftDelete();
-
       // Reload iframe so we don't leave users without a reply UI.
-      window.location.reload();
+      this.v.skipClickPrompt = false;
+      window.location.href = Env.urlCreate(window.location.protocol + '//' + window.location.hostname + window.location.pathname, this.v);
```

Comment:
You could use `window.location.href.split('?')[0]` which is shorter but your is cleaner so it's just a matter of taste. Looks good.